packages/ui/react-ui-table/src/model/table-sorting.ts

```diff
@@ -38,13 +34,12 @@ export type SortConfig = { fieldId: string; direction: SortDirection };
  */
 export class TableSorting<T extends BaseTableRow> {
   private readonly _displayToDataIndex = new Map<number, number>();
-  private readonly _sorting = signal<SortConfig | undefined>(undefined);
   private readonly _rows: Signal<T[]>;
   public readonly sortedRows: ReadonlySignal<T[]>;
 
   constructor(
     rows: Signal<T[]>,
-    private readonly _table: TableType,
+    private readonly _view: ViewType | undefined,
```

Comment:
is this really optional? if so it should probably be the last arg?

Reply:
The view is optional on table type so that propagates to here.